CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Breaking changes:
 New features:
 
 Bugfixes:
+- Added missing type class instance names for `DataTransferItemKind` (#69 by @ptrfrncsmrph)
```

Comment:
Seemed like **Bugfixes** was the most appropriate category, let me know if I should move it somewhere else